ui/src/main/java/org/eclipse/starter/ui/Project.java

```diff
@@ -143,10 +151,23 @@ public void setRuntime(String runtime) {
 		this.runtime = runtime;
 	}
 
+	public String getGroupId() {
+		return groupId;
+	}
+
+	public void setGroupId(String groupId) {
+		this.groupId = groupId;
+	}
+
+	public String getArtifactId() {
+		return artifactId;
+	}
+
+	public void setArtifactId(String artifactId) {
+		this.artifactId = artifactId;
+	}
+
 	public void onJakartaVersionChange() {
-		LOGGER.log(Level.INFO,
```

Comment:
Please restore these log entries. They are important to understand what is going on in the application. If log volume is a concern (but quite honestly current log volumes are very modest), the log level can be downgraded a bit.